android/src/main/java/de/atroo/securestorage/EncryptedPreferencesImpl.java

```diff
@@ -0,0 +1,81 @@
+package de.atroo.securestorage;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+import androidx.security.crypto.EncryptedSharedPreferences;
+import androidx.security.crypto.MasterKeys;
+
+import java.util.Map;
+import java.util.Set;
+
+@RequiresApi(api = Build.VERSION_CODES.M)
+public class EncryptedPreferencesImpl implements SecureStorageInterface {
+    private static final String TAG = "EncryptedPrefsImpl";
+    private SharedPreferences sharedPreferences;
+
+    @Override
+    public boolean init(Context context, String preferencesName) {
+        try {
+            String masterKeyAlias = MasterKeys.getOrCreate(MasterKeys.AES256_GCM_SPEC);
+
+            sharedPreferences = EncryptedSharedPreferences.create(
```

Comment:
@laszloblum The docs (https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences) mention to specify backup rules for exlcuding the keys. Is this something we should do in the plugin or create a piece of documentation for the usage?

Reply:
No, it should be done in the app's AndroidManifest.xml
I have added it to the README.